pkg/modprobe/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM linuxkit/alpine:87a0cd10449d72f374f950004467737dbf440630 as mirror
+
+RUN mkdir -p /out/etc/apk && \
+    cp -r /etc/apk/* /out/etc/apk/ && \
+    apk add --no-cache --initdb -p /out musl && \
+    rm -rf /out/etc/apk /out/lib/apk /out/var/cache && \
+    mkdir /out/bin && \
+    mkdir /out/sbin && \
+    cp /bin/busybox /out/bin/busybox && \
+    cp -P /sbin/modprobe /out/sbin/modprobe
```

Comment:
@rn, I've cleaned up the `pkg/modprobe/Dockerfile` here per your preferred style (which is my actually my preferred style as well). I was simply trying to keep the existing pattern from the other included `pkg`; I think you'll find the single line `&&` style is common in a lot of the existing `pkg/*/Dockerfile`

Reply:
Argh, I think we misunderstood...the previous version had one line with multiple commands in it and I was proposing to just split that line up. something like:
```
RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
RUN apk add --no-cache --initdb -p /out \
    alpine-baselayout \
    busybox \
    musl

RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
```

You don't actually need to copy `cp /bin/busybox /out/bin/busybox` and `cp -P /sbin/modprobe /out/sbin/modprobe` as the `-p /out` argument to `apk` installs the packages directly in `/out`. So what you are doing is copy the `busybox` binary from `linuxkit/alpine:87a0cd10449d72f374f950004467737dbf440630` to `/out` insteda of the one `apk` installs. (they should be same).